bciers/apps/administration/app/components/operators/getOperator.ts

```diff
@@ -2,12 +2,15 @@ import { actionHandler } from "@bciers/actions";
 import { UUID } from "crypto";
 
 // 🛠️ Function to get an operator by operator id
-export default async function getOperator(id: UUID) {
+export default async function getOperator(
+  id: UUID,
+  pathToRevalidate: string = "",
+) {
   try {
```

Comment:
I think we decided we don't need try/catch: https://teams.microsoft.com/l/message/19:81544b60877c4a5eb4fc7946ecbcb525@thread.tacv2/1729182397476?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=29821293-3244-455d-beae-e01d27762cfe&parentMessageId=1729182397476&teamName=External%3A%20CAS%20-%20Clean%20Growth%20Digital%20Service%20Team&channelName=Developers&createdTime=1729182397476&ngc=true

Reply:
This will be handled by the rebase.